rs/tests/ict/cmd/testnetCreateCmd.go

```diff
@@ -183,6 +183,21 @@ func TestnetCommand(cfg *TestnetConfig) func(cmd *cobra.Command, args []string)
 		}
 		command := []string{"bazel", "test", target, "--config=systest"}
 		command = append(command, "--cache_test_results=no")
+		icDashboardsDir, err := sparse_checkout("git@github.com:dfinity-ops/k8s.git", "", []string{"bases/apps/ic-dashboards"})
```

Comment:
it looks like the repo is wiped and checked out again on every run: https://github.com/dfinity/ic/pull/2798/files?file-filters%5B%5D=.bazel&file-filters%5B%5D=.bzl&file-filters%5B%5D=.go&file-filters%5B%5D=.md&file-filters%5B%5D=.rs&file-filters%5B%5D=No+extension&show-deleted-files=true&show-viewed-files=true#diff-2461b3fd6db4364e451070192845ecf1597a43dc43cb2eb41da8d729fc73bb90R169

would it make sense to keep a cached checkout? alternatively, if we do want to re-check it out every time, why not check it out to a tmp dir?

Reply:
I didn't exactly measure the time difference, but we would have to either `pull` or do the sparse checkout config on each run. The time difference isn't huge, but in theory I could make it to just pull the data if you think it would be better